python/py-openssl/Portfile

```diff
@@ -45,13 +48,15 @@ if {${name} ne ${subport}} {
                 ${destroot}${prefix}/share/doc/${subport}
     }
 
-    if {${python.version} == 27} {
+    if {${python.version} == 27 \
+        || ${os.platform} eq "darwin" && ${os.major} < ${openssl_darwin_min_ver}} {
+        # TODO: see if we can upgrade this, reverting commits which rely on Rust.
         github.setup    pyca pyopenssl 21.0.0
         checksums       rmd160  5806276d4716458a77c32f91a0c7f8ee30fe1dfb \
                         sha256  5d1d93fb3d55be740e444b991bccc5db26b1cfd3666cbbd8b6d01dc7a6cc5430 \
                         size    172965
         revision        0
-        epoch           1
+        epoch           2
```

Comment:
I don't think this should increase the "epoch"; it was already correct for the PY27 subport and having it set to 1 for older versions now effectively increased the epoch for those as well.

Reply:
Thank you, I will fix it now.

Reply:
@reneeotten Done